File: common/src/main/java/com/common/entities/potions/HP.java
package com.common.entities.potions;

import com.common.DataClasses.Position;
import com.common.Debug;
import com.common.entities.templates.Potion;
import com.common.graphics.Icons;

import static com.common.Debug.Levels.CORE;
import static com.common.Debug.debug;

/**
 * Health potion
 */
public class HP extends Potion {
	public HP(Position position) {
		super(position);
		debug(Debug.Flags.Entities.Potions.HP, CORE, "--- [HP.constructor]");
		icon = Icons.Interactive.hp_potion;
		heal = 10;
	}
}

File: composeApp/src/commonMain/kotlin/com/one_of_many_simons/one_button_game/entities/potions/HP.kt
package com.one_of_many_simons.one_button_game.entities.potions

import com.one_of_many_simons.one_button_game.Debug.Flags.Entities.Potions
import com.one_of_many_simons.one_button_game.Debug.Levels.CORE
import com.one_of_many_simons.one_button_game.Debug.debug
import com.one_of_many_simons.one_button_game.dataClasses.Position
import com.one_of_many_simons.one_button_game.entities.templates.Potion
import com.one_of_many_simons.one_button_game.graphics.Icons

/**
 * Health potion
 */
class HP(position: Position) : Potion(position) {
    init {
        debug(Potions.HP, CORE, "--- [HP.constructor]")
        icon = Icons.Interactive.hp_potion
        heal = 10
    }
}
